rooted-ota.sh

```diff
@@ -64,7 +64,8 @@ UPLOAD_TEST_OTA=${UPLOAD_TEST_OTA:-false}
 
 OTA_CHANNEL=${OTA_CHANNEL:-stable} # Alternative: 'alpha'
 NO_COLOR=${NO_COLOR:-''}
-OTA_BASE_URL="https://releases.grapheneos.org"
+OTA_BASE_URL=${OTA_BASE_URL:-"https://releases.grapheneos.org"} # For LineageOS: https://download.lineageos.org/api/v2
+GRAPHENE_TYPE=${GRAPHENE_TYPE-'ota_update'} # Other option: factory
```

Comment:
The `:` is not missing here, to [allow](https://www.baeldung.com/linux/bash-variable-assign-default) `GRAPHENE_TYPE` being empty.